modules/navier_stokes/src/functormaterials/INSFVEnthalpyFunctorMaterial.C

```diff
@@ -32,31 +32,35 @@ INSFVEnthalpyFunctorMaterial::validParams()
       NS::enthalpy_density, NS::enthalpy_density, "the name of the (extensive) enthalpy");
   params.addParam<MooseFunctorName>(
       NS::specific_enthalpy, NS::specific_enthalpy, "the name of the specific enthalpy");
+  params.addParam<std::string>(
+      "property_suffix", "", "Suffix for disambiguating functor property names");
```

Comment:
```suggestion
      "property_suffix", "", "Suffix to append to declared functor property names for global property disambiguation");
```

Reply:
want to make sure there is a clear delineation between this parameter and the `prop_getter_suffix` that @grmnptr just enabled for functor materials

Reply:
Oh wait. In @grmnptr's https://github.com/idaholab/moose/pull/27442 we are using the `prop_getter_suffix` parameter to modify the *declared* property name? That was not the original intent of that parameter and it's contrary to the name of the parameter. I approved it too 😦 There is actually a `declare_suffix` parameter for materials that is explicitly meant for this. But I see that @GiudGiud suppressed this parameter in idaholab/moose@8d0f9c8a39aa91f530384e95c1c86a45b6a10f09

Reply:
I suppressed it because it was silently being ignored.
 
do you want me to re-enable it for all functors then? and remove this one

Reply:
Yea it would be good to unsuppress it and I would like for the code/test from #27442 to use the `declare_suffix` instead of the `prop_getter_suffix` and then remove this parameter

Reply:
done